main/Button/stories/DefaultStory.tsx

```diff
@@ -7,6 +6,8 @@ import styled, {css} from '@emotion/native';
 
 import type {FC} from 'react';
 import {action} from '@storybook/addon-actions';
+import {Button} from '..';
+import {Hr} from '../../Hr';
```

Comment:
I think the prev changes are better since we know that the components all lie on the same level.